Scripts/CommonServerPython/CommonServerPython.md

```diff
@@ -1,2 +1,3 @@
 ## [Unreleased]
```

Comment:
this is the wrong file, please move them to `Scripts/CommonServerPython/CHANGELOG.md` and delete this file.